sos/report/plugins/ceph_iscsi.py

```diff
@@ -0,0 +1,50 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin, UbuntuPlugin
+
+
+class CephISCSI(Plugin, RedHatPlugin, UbuntuPlugin):
+
+    short_desc = "CEPH iSCSI"
+    plugin_name = "ceph_iscsi"
+    profiles = ("storage", "virt")
+
+    packages = ("ceph-iscsi",)
+    services = ("rbd-target-api", "rbd-target-gw")
+
+    def setup(self):
+        self.add_copy_spec(
+            [
+                "/etc/tcmu/tcmu.conf",
+                "/var/log/ceph/ceph-client.iscsi.log",
+                "/var/log/rbd-target-api/rbd-target-api.log",
+                "/var/log/rbd-target-gw/rbd-target-gw.log",
+                "/var/log/tcmu-runner.log",
+            ]
+        )
```

Comment:
Minor nitpick - sos style is to have the list brackets in-line with the parens of the calling method, thus reducing the indention level by one. E.G.:

```python
self.add_copy_spec([
    '/some/path',
    '/some/other/path'
])
```

Reply:
Done. It was from black; the format has been updated.